doc/meetings/2023_04_04.md

```diff
@@ -0,0 +1,99 @@
+# Meeting 2023-04-04
+
+_Protocolled by: Nursultan_
+
+## Attendees
+
+-   Benjamin (Best PermaplanT Companion KW13)
+-   Nursultan (Best PermaplanT Companion KW12)
+-   Giancarlo (Best PermaplanT Companion KW11)
+-   Paul
+-   Samuel
+-   Gabriel
+-   Moritz
+-   Lukas
+-   Ramzan
+-   Thorben
+-   Markus
+-   Yvonne
+
+## Buddies
+
+-   Giancarlo & Moritz
+-   Lukas & Gabriel
+-   Samuel & Nursultan
+-   Ramzan & Thorben
+-   Benjamin & Paul
+
+## Tasks for Everyone
+
+To be done within this week (31.03.2023):
+
+-   Finalize [use cases](https://github.com/ElektraInitiative/PermaplanT/issues/1)
+-   Read all decisions, progress report which ones you agree
+-   Start decisions [#2](https://github.com/ElektraInitiative/PermaplanT/issues/2)
+-   Write Forschungstagebuch in submissions repo (longer progress report)
```

Comment:
@markus2330 What do you mean by Forschungstagebuch? How should that look like?

Just a .txt like the following?
```
29.03.2023 (2h): latex template
30.03.2023 (2h): thesis structure
...
```

Reply:
Format is as you like, basically you can also write by hand and only give short summaries in the weekly notes. The idea is from Judith Wolfsberger "Frei geschrieben" https://www.freigeschrieben.at/